README.md

```diff
@@ -1,4 +1,104 @@
 # KindedJ
 A collection of shared interfaces for evidence-based Higher Kinded Types in the JVM
 
-Gitter: https://gitter.im/KindedJ/Lobby
+[![Gitter chat](https://badges.gitter.im/KindedJ/KindedJ.png)](https://gitter.im/KindedJ/Lobby)
+
+## Projects that support KindedJ
+
+Name & Organisation - Language - Supported since version
+
+#### List
+
+*Add yourself here by [sending a Pull Request](https://github.com/KindedJ/KindedJ/compare)*
+
+[Kategory](https://github.com/kategory/kategory) - Kotlin - 0.3.6
+
+## Purpose
+
+A Higher Kinded Type represents a value that wraps another value. It would be equivalent to F\<A> if Java supported higher kinds. This allows library writers to create abstractions that work on any wrapper type in a completely generic fashion. You can see some examples in the projects supporting KindedJ listed above.
```

Comment:
I'd add some reference article, perhaps wikipedia here. Also I'd add that a higher kind is a contructor of types and perhaps add simple example datatype for those that land here and are not familiar with the concept.